src/@next/components/atoms/RichTextEditorContent/RichTextEditorContent.tsx

```diff
@@ -48,6 +48,10 @@ export const getTools = async (): Promise<
 export const RichTextEditorContent: React.FC<RichTextEditorContentProps> = ({
   jsonData,
 }) => {
+  if (!jsonData) {
```

Comment:
Return statements should come after all hook declarations as this guarantees their call order during render phase:

https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level